src/node_options-inl.h

```diff
@@ -388,18 +388,18 @@ void OptionsParser<Options>::Parse(
     }
 
     {
-      std::string implied_name = name;
-      if (is_negation) {
-        // Implications for negated options are defined with "--no-".
-        implied_name.insert(2, "no-");
-      }
-      auto implications = implications_.equal_range(implied_name);
```

Comment:
Before the fix, there was likely code intended to apply negations to implications as well, but options with the --no- prefix were never added to the implications_.